src/content/docs/tutorial-errors/solve-critical-errors.mdx

```diff
@@ -0,0 +1,105 @@
+
+import apmLogsZoom from 'images/apm_screenshot-crop_errors-logs-zoomed.webp'
+
+import apmStackTraces from 'images/apm_screenshot-crop_errors-stack-traces.webp'
+
+import apmErrorIDE from 'images/apm_screenshot-crop_errors-IDE.webp'
+
+Once you've chosen your error group, New Relic takes you to a new page that displays attributes, logs, and traces about the error. This part of the tutorial guides you through two recommended avenues for analyzing an error, your logs and stack traces. 
+
+## Objectives [#objectives]
+
+This doc builds off the problem scoped from the [Respond to service outages](/docs/tutorial-errors/respond-outages) doc of this tutorial series. In this doc, you'll:
+
+* Locate the error code associated with a failing API with logs
+* Use stack traces to find where in your code the error is occurring
+
+## Locate the source of an error with logs and stack traces [#analyze]
+
+The errors summary tells the story behind your error groups. Depending on how you've configured New Relic, each service might display different kinds of information about an error. For example, disabling distributed tracing limits trace details about external services.
+
+<Steps>
+    <Step>
+
+    ### Open the Log details window
+
+Looking at our example, the APM agent has pulled logs related to `api-gateway`, giving us event data about our error:
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmLogs}
+    />
+
+Our logs in context capability formats your logs information, though you have the option to investigate the unformatted log as well. 
+    </Step>
+    <Step>
+
+    ### Find the failure point in your system
+        
+Depending on the nature of the log, you may have more or less logging detail to sort through. Because you spent time upfront narrowing down the service closest to the failure and choosing the likely error group, you have time to read your logs. 
+
+<img
+    title="Overview errors affecting your services"
+    alt="A screenshot showing an app with many errors"
+    src={apmLogsZoom}
+    />    
+
+According to your logs, your problem is:
+
+* A timeout error: `error.error.code: ETIMEDOUT`
+* Related to your customer API: `error.endpoint: customers-api-internal`
+* Agnostic to the kind of request sent to the customer API: `error.request: /api/customers/search/Kirlin/-/`
+
+You have enough context to tie it together. You conclude that there's a dependency failure: All calls to the customer API are timing out and causing upstream requests from `api-gateway` to fail. 
+
+</Step>
+
+<Step>
+
+    ### Find the failure point in your code with stack traces
+
+When you run your code, New Relic captures your exceptions or runtime errors and organizes them into a waterfall view. These are your stack traces. Depending on the language used, these stack traces provide detailed messages and the locations of your failure points.  
```

Comment:
> Depending on the language used, these stack traces provide detailed messages and the locations of your failure points.  

Is this trying to say that sometimes we get this data and sometimes we don't, depending on which APM language agent you're using? Or is it trying to say that the exact data we gather varies based on language, but we always collect things like messages and locations